package.json

```diff
@@ -25,6 +25,7 @@
     "@sap/cds-foss": ">=5"
   },
   "dependencies": {
+    "inquirer": "^9.2.20",
```

Comment:
We don't seem to need/use inquirer - can you get rid of it?

Reply:
Removed